whitesource/util.py

```diff
@@ -1,10 +1,167 @@
 import functools
 
+import tabulate
+import typing
+
 import ci.util
 import whitesource.client
+import mailutil
+import mail.model
 
 
 @functools.lru_cache()
 def create_whitesource_client(whitesource_cfg_name: str):
     cfg_fac = ci.util.ctx().cfg_factory()
     return whitesource.client.WhitesourceClient(cfg_fac.whitesource(whitesource_cfg_name))
+
+
+def generate_reporting_tables(report,
+                              threshold: float,
+                              tablefmt):
+
+    # monkeypatch: disable html escaping
+    tabulate.htmlescape = lambda x: x
+
+    # split respecting CVSS-V3 threshold
+    above = {}
+    below = {}
+
+    for lib, dic in report.items():
+        if float(dic['CVSS-V3']) > threshold:
+            above[lib] = dic
+        else:
+            below[lib] = dic
```

Comment:
btw, this approach has the disadvantage that conflicts will not be detected (equally-named libs..)
also, I would prefer to use more expressive names, e.g. `lib_name`, i.p. seeing that the report is simply a dict (is that the case?)

Reply:
as a general "rule-of-thumb", I think that if using less strict typing, at least more expressive names / naming conventions ought to be used